src/themes/eui-amsterdam/global_styling/index.scss

```
@@ -0,0 +1,16 @@
+// Core
+
+// Functions need to be first, since we use them in our variables and mixin definitions
+@import '../../../global_styling/functions/index';
+
+// Variables come next, and are used in some mixins
+@import '../../../global_styling/variables/index';
+
+// Mixins provide generic code expansion through helpers
+@import 'mixins/index';
```

This can be simplified by purely providing the original global index and then overriding with your own import.
```suggestion
@import '../../../global_styling/mixins/index';
@import 'mixins/index';
```